charts/app-config/values-integration.yaml

```diff
@@ -1721,7 +1721,7 @@ govukApplications:
           ]}}]
       hosts:
         - name: release.{{ .Values.k8sExternalDomainSuffix }}
-    workerEnabled: false
+    workerEnabled: true
```

Comment:
optional: do you need the worker in integration and staging, if it's not posting to Slack from there?

Reply:
yeah, good point. Will remove. Added it for consistency across environments so we have an option to test or debug but may be better to saves the resources.

Reply:
Cool, SGTM. Consistency can sometimes be helpful but sounds like probably not in this case 👍 Thanks for being efficient with resources!